test/bootstrap.jl

```diff
@@ -65,4 +98,26 @@ end
         @test sort(columntable(bsamp_threaded.β).β) == sort(columntable(bsamp.β).β)
         @test sum(issingular(bsamp)) == sum(issingular(bsamp_threaded))
     end
+
+
+    @testset "Bernoulli simulate! and GLMM boostrap" begin
+        contra = dataset(:contra)
+        gm0 = fit(MixedModel, only(gfms[:contra]), contra, Bernoulli(), fast=true)
+        bs = parametricbootstrap(StableRNG(42), 100, gm0)
+        bsci = combine(groupby(DataFrame(bs.β), :coefname),
+                       :β => first ∘ shortestcovint => :lower,
+                       :β => last ∘ shortestcovint => :upper)
```

Comment:
Its a very minor gripe but `shortestcovint` is being called twice on the same bootstrap results and it is comparatively expensive (extract the data, sort it, calculate the end points, run the loop to determine the shortest interval, etc.). I would just store the intervals and do the calls to `first` and `last` later when I need to extract the endpoints.

Reply:
Fair point and something I meant to clean up before pushing, but then forgot.